conode/Makefile

```diff
@@ -11,7 +11,8 @@ cothority:
 conode: cothority
 	@( cd cothority; git pull )
 	echo "Compiling binaries"; \
-	GO111MODULE=on go build -o build/conode; \
+	export GO111MODULE=on; \
```

Comment:
So, I saw a lot of `GO111MODULE` defined around; from the [golang wiki](https://github.com/golang/go/wiki/Modules#how-to-install-and-activate-module-support), it seems that it's not needed in our cases.
And so, I've encountered the same issue as you, if one of its deps is not a module, it fails (for example with https://github.com/qantik/qrgo); maybe fixing upstream (it's from someone who was at DEDIS)?

Reply:
True - IIRC, if the .go file is not in the `$GOPATH/src/$(head -n 1 go.mod)` the module should be activated anyway.

But usually the qantik-repo should be included correctly in the `go.mod` file, so I don't really understand why it fails here?

Reply:
> But usually the qantik-repo should be included correctly in the go.mod file, so I don't really understand why it fails here?

I don't know either, maybe a missing `go.mod` in the repo? That's the only guess I have, you're the go master here :)